### PR TITLE
Add new files for uk market conformity assessment bodies

### PR DESCRIPTION
**Pull request by Seraphiana**
PR 3 of 3 for this work. This updates the content schemas to have a new specialist document type, uk market conformity assessment bodies.

### CODE REVIEW

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 1]
```diff
@@ -412,6 +415,287 @@
...
+      },
+      uk_market_conformity_assessment_body_registered_office_location: {
+        type: "string",
+        enum: [
+          "united-kingdom",
+          "japan",
+          "switzerland",
+          "united-states-of-america",
+          "australia",
+          "new-zeeland",
```
Comment (sihugh) [inline formats/shared/definitions/_specialist_document.jsonnet]: Possibly a dodgy spelling?

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 2]
```diff
@@ -412,6 +415,287 @@
...
+            "recognised-third-party-organisation",
+            "technical-assessment-body",
+            "user-inspectorate",
+          ],
+        },
+      },
+      uk_market_conformity_assessment_body_registered_office_location: {
+        type: "string",
+        enum: [
+          "united-kingdom",
```
Comment (sihugh) [inline formats/shared/definitions/_specialist_document.jsonnet]: The order of these seems confusing. Worth clarifying whether this is desired?

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 3]
```diff
@@ -412,6 +415,287 @@
...
+          "israel",
+          "canada",
+        ],
+      },
+      uk_market_conformity_assessment_body_testing_locations: {
+        type: "array",
+        items: {
+          type: "string",
+          enum: [
+            "north-macedonia",
```
Comment (sihugh) [inline formats/shared/definitions/_specialist_document.jsonnet]: Same comment about confusing order.

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 4]
```diff
@@ -412,6 +415,287 @@
...
+            "papua-new-guinea",
+            "panama",
+            "palau",
+            "pakistan",
+            "oman",
+            "norway",
+            "nigeria",
+            "niger",
+            "nicaragua",
+            "new-zealand",
```
Comment (sihugh) [inline formats/shared/definitions/_specialist_document.jsonnet]: This one seems OK :-)

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 5]
```diff
@@ -412,6 +415,287 @@
...
+      },
+      uk_market_conformity_assessment_body_phone: {
+        type: "string",
+      },
+      uk_market_conformity_assessment_body_legislative_area: {
+        type: "array",
+        items: {
+          type: "string",
+          enum: [
+            "toys",
```
Comment (sihugh) [inline formats/shared/definitions/_specialist_document.jsonnet]: ordery thing again

File: formats/shared/definitions/_specialist_document.jsonnet (hunk 1 above)
Comment (Seraphiana) [reply formats/shared/definitions/_specialist_document.jsonnet]: probably so?

File: formats/shared/definitions/_specialist_document.jsonnet (hunk 3 above)
Comment (Seraphiana) [reply formats/shared/definitions/_specialist_document.jsonnet]: I've updated the order. Looks like it was alphabetical with UK at the top? originally? that's what I've gone with at least

File: formats/shared/definitions/_specialist_document.jsonnet (hunk 2 above)
Comment (Seraphiana) [reply formats/shared/definitions/_specialist_document.jsonnet]: updated

**Pull request merged**